rpc/eth/jsonrpc_conversion.go

```diff
@@ -406,6 +406,9 @@ func DecDataToBytes(value Data) ([]byte, error) {
 	if len(value) <= 2 || value[0:2] != "0x" {
 		return []byte{}, errors.Errorf("invalid data format: %v", value)
 	}
+	if len(value)%2 != 0 {
+		value = "0x0" + value[2:]
+	}
```

Comment:
According to the web3 JSON-RPC spec the UNFORMATTED DATA value must always have an even number of digits https://github.com/ethereum/wiki/wiki/JSON-RPC#hex-value-encoding so if `len(value) % 2 != 0` we should return an error.

Reply:
Umm, `0x0` is a valid param though. Take a look at eth_getStorageAt example here. https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_getstorageat

Reply:
Good point, `DecDataToBytes` was actually the wrong thing to use. See https://github.com/loomnetwork/loomchain/pull/1449#discussion_r323075687